recipes/r-xcell/meta.yaml

```diff
@@ -0,0 +1,62 @@
+package:
+  name: r-xcell
+  version: "1.2"
+
+source:
+  url:
+    - https://github.com/dviraran/xCell/archive/1.2.tar.gz
+  sha256: 52dcd5cfb5c8a4870cb485ee57f34d7830b23bdc61fd3012d859fdcc22ad941c
+
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+
+  host:
+    - r-base =3.5.1
```

Comment:
please do not pin r-base and also the other pins do look way to strict

Reply:
done. 
As the original DESCRIPTION does not specify any version, I removed all versions here, too.